bin/webpack.js

```diff
@@ -1,19 +1,43 @@
 #!/usr/bin/env node
 
+const { exec } = require("child_process");
+const readline = require("readline");
+
+let io = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
 let webpackCliInstalled = false;
+
 try {
-	require.resolve("webpack-cli");
+	require.resolve("webpack-cli") &&
+	require("webpack-cli");
+} catch(err) {
 	webpackCliInstalled = true;
-} catch(e) {
-	webpackCliInstalled = false;
+	process.stdout.write("Sorry, 'webpack-cli' was not found inside this project.");
+
+	io.question("Do you want to install 'webpack-cli' automatically? [ Y/N ] \n", answer => {
```

Comment:
What happens when running this in CI. Is there a way to check if this is an interactive terminal?

Reply:
node ./bin/webpack.js ?